multichain-testing/test/xcs-swap-anything/Makefile

```
@@ -0,0 +1,108 @@
+cli-hermes=kubectl exec -i hermes-agoric-osmosis-0 -c relayer -- hermes
+exec-agd=kubectl exec -i agoriclocal-genesis-0 -c validator -- agd
+exec-osmo=kubectl exec -i osmosislocal-genesis-0 -c validator -- osmosisd
+cli-osmo=kubectl exec -i osmosislocal-genesis-0 -c validator -- /bin/bash -c
+
+AGORIC_TEST_ADDRESS=$(shell $(exec-agd) keys show test1 -a --keyring-backend test)
+AGORIC_GENESIS_ADDRESS=$(shell $(exec-agd) keys show genesis -a --keyring-backend test)
+OSMOSIS_TEST_ADDRESS=$(shell $(exec-osmo) keys show test1 -a --keyring-backend test)
+OSMOSIS_GENESIS_ADDRESS = $(shell $(exec-osmo) keys show genesis -a --keyring-backend test)
+
+TX_FLAGS=--from $(OSMOSIS_GENESIS_ADDRESS) --keyring-backend=test --gas=auto --gas-prices 0.1uosmo --gas-adjustment 1.3 --yes --chain-id osmosislocal
+AGD_TX_FLAGS=--from $(AGORIC_GENESIS_ADDRESS) --keyring-backend=test --gas=auto --gas-prices 0.1ubld --gas-adjustment 1.3 --yes --chain-id agoriclocal
```

suggest: make it clearer which parts are the same by moving them to the beginning and making the variable names the same length:

```suggestion
OSM_TX_FLAGS=--keyring-backend=test --gas=auto --gas-adjustment 1.3 --yes --from $(OSMOSIS_GENESIS_ADDRESS) --gas-prices 0.1uosmo --chain-id osmosislocal
AGD_TX_FLAGS=--keyring-backend=test --gas=auto --gas-adjustment 1.3 --yes --from $(AGORIC_GENESIS_ADDRESS) --gas-prices 0.1ubld --chain-id agoriclocal
```

If this were in `.js`, it would be straightforward to reify the authority wielded by these accounts as js objects. I haven't landed it yet, but some work in progress...

https://github.com/Agoric/agoric-sdk/blob/9420b74a9d095ab1afb19945597304022cb07c9f/multichain-testing/tools/agd-lib.js#L179

---

This Makefile was removed, and its logic was implemented on `multichain-testing/test/xcs-swap-anything/helpers.ts`

See commit: 65732310e650ab03d058877b33b2982112a95d21